src/components/commons/prismicTypes.ts

```diff
@@ -1,7 +1,14 @@
 export interface Posts {
-  title: {
-    text: string;
-  };
+  title: [
+    {
+      text: string;
```

Comment:
If we define the type this way, the array will only contain one object

Reply:
yes, it will only contain one!